px4_ros2_cpp/include/px4_ros2/navigation/experimental/global_navigation_interface.hpp

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+
+#pragma once
+
+#include <optional>
+#include <rclcpp/rclcpp.hpp>
+#include <Eigen/Eigen>
+
+#include <px4_msgs/msg/vehicle_global_position.hpp>
```

Comment:
We should add the topics used to the message compatibility checks. See https://github.com/Auterion/px4-ros2-interface-lib/blob/main/px4_ros2_cpp/include/px4_ros2/components/message_compatibility_check.hpp#L12-L33.